### PR TITLE
Terra profile image nightwatch  --> wdio

### PR DESCRIPTION
**Pull request by msaikaushik1992**
### Summary
Resolves [this](https://github.com/cerner/terra-core/issues/1302).

### CODE REVIEW

File: packages/terra-profile-image/tests/wdio/profile-image-spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* global browser, Terra, before */
+
+describe('Profile Image', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/profile-image/profile-image-default'));
+
+    Terra.should.beAccessible();
```
Comment (emilyrohrbough) [inline packages/terra-profile-image/tests/wdio/profile-image-spec.js]: can we break this into two snapshots, one for default and one for error?

File: packages/terra-profile-image/examples/test-examples/ProfileImageDefault.example.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ProfileImage from '../../lib/ProfileImage';
+import exampleProfileImage from '../../tests/assets/150x150.jpg';
+
+const ProfileImageDefault = () => (
+  <div>
+    <h2>Successful Profile Image</h2>
+    <ProfileImage src={exampleProfileImage} alt="proflie image loaded" width="75" height="75" />
```
Comment (emilyrohrbough) [inline packages/terra-profile-image/examples/test-examples/ProfileImageDefault.example.jsx]: whoops, type in alt '~proflie~ **profile** image loaded'

File: packages/terra-profile-image/examples/test-examples/ProfileImageDefault.example.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <br />
+    <h2>Failed Profile Image</h2>
+    <ProfileImage src="invalid.jpg" alt="could not load proflie image" width="75" height="75" />
```
Comment (emilyrohrbough) [inline packages/terra-profile-image/examples/test-examples/ProfileImageDefault.example.jsx]: whoops, type in alt 'proflie profile image loaded'

File: packages/terra-profile-image/tests/wdio/profile-image-spec.js (hunk 1 above)
Comment (msaikaushik1992) [reply packages/terra-profile-image/tests/wdio/profile-image-spec.js]: [updated](https://github.com/cerner/terra-core/pull/1500/commits/d6fcb7b7f69779567facda2b3748da26150bc30f).

**Pull request merged**